README.md

```diff
@@ -116,6 +116,26 @@ The request body is a json array where each entry of the array is an object to s
   parameter is optional, and Autograph will pick a key based on the caller's
   permission if omitted.
 
+* signature_encoding: by default, signatures returned by autograph use a R||S
+  string format encoded with base64_urlsafe. The R||S format simply concatenates
+  the two integer value that compose an ECDSA signature into one big number
+  (for p384, each value is 48 bytes long, so the total is 96 bytes). This format
+  avoid relying on ASN.1 parser to read the signatures, but can make it difficult
+  to verify signatures without custom code. The base64_urlsafe encoding format
+  strips base64 padding and replaces characters  `+` and `/` with `-` and `_`
+  respectively.
+  The R||S base64_urlsafe format complies with the Content Signature protocol,
```

Comment:
I don't know what the "Content Signature protocol" is. Seems like a good place to add a link :)